### PR TITLE
add section for SRA caveats

### LINKED ISSUE #7287: Secure Runtime Access Doc Update
Re: [Secure Runtime Access](https://pantheon.io/docs/secure-runtime-access)

Priority: Low

## Issue Description:

Users with Secure Runtime Access need to have an active Dashboard session in order to access DB services.

## Suggested Resolution

from @ryanshoover 

> - You can't access mysql, or sftp unless you have an active dashboard session. Terminus may be blocked as well, but i'm not 100% on that.
> - Dashboard sessions only last 24 hours (consistent with [user dashboard doc](https://pantheon.io/docs/user-dashboard#pantheon-user-account-login-session-length)
> - You can start a new dashboard session by logging into [dashboard.pantheon.io](http://dashboard.pantheon.io/) or running terminus auth:login
> - We determine which user should have access to mysql, sftp through the ssh key being used and which user account has the public key
> - CI/CD integrations on sites that have SRA enabled and that rely on any of these services need to ensure they're creating a dashboard session before trying to their thing

### PR DESCRIPTION
**Pull request by joa-pan**
Closes #7287 

## Summary

**[Secure Runtime Access](https://pantheon.io/docs/secure-runtime-access)** -  Add considerations because users with Secure Runtime Access need to have an active Dashboard session in order to access DB services.

## Effect

The following changes are already committed:

*
*

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] List any outstanding work here

### Dependencies and Timing

- [ ] Other prerequisites that must be completed before merging this PR

**Release**:
- [ ] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Review by ryanshoover — COMMENTED** [review]
I believe this is accurate, but I'm not sure about the restrictions. If memory serves, MySQL and SFTP are protected but Redis isn't. I think with MySQL there's also a caveat that, with SRA, connecting to the database is protected but you can still reach the server.

@Hornswoggles Any chance you can verify the information here?

**Review by Hornswoggles — APPROVED** [review]
+1

File: source/content/secure-runtime-access.md [hunk 1]
```diff
@@ -17,6 +17,14 @@ In addition to defense in depth, this feature can be used to enforce role-based
 
 Secure Runtime Access is available to [contract](https://pantheon.io/plans/pricing) customers with an [Organization](/organizations) dashboard. [Contact Sales](https://pantheon.io/contact-us) to request that SRA be enabled for your site.
 
+### Considersations
+
+Users with Secure Runtime Access must have an active dashboard session to access database services. The following considerations apply:
```
Comment (EdwardAngert) [inline source/content/secure-runtime-access.md:22]: is logging in to the Pantheon dashboard via the browser the only way to start `an active dashboard session`?

File: source/content/guides/secure-development/07-secure-runtime-access.md [hunk 2]
```diff
@@ -15,9 +15,9 @@ reviewed: "2022-07-21"
 
 This section provides information on how you can use Secure Runtime Access (SRA) with SSH tunnels to keep your sites secure.
 
-Pantheon’s database services use strong, random passwords and TLS to encrypt communications by default. Customers seeking additional defense can enable Secure Runtime Access.
+Pantheon’s database services use strong, random passwords and TLS to encrypt communications by default. Customers seeking additional defense can enable Secure Runtime Access (SRA).
 
-SRA actively discards attempts to connect to persistent instances like MySQL or SFTP, disregarding the attempt before it reaches the service. When SRA is enabled, the connection attempts to the service will be rejected unless the connection comes through the appropriate [SSH tunnel](/guides/secure-development/ssh-tunnels).
+SRA actively discards attempts to connect to persistent instances like MySQL databases or SFTP, disregarding the attempt before it reaches the service. When SRA is enabled, the connection attempts to the service will be rejected unless the connection comes through the appropriate [SSH tunnel](/guides/secure-development/ssh-tunnels).
```
Comment (EdwardAngert) [inline source/content/guides/secure-development/07-secure-runtime-access.md:20]: @ryanshoover @Hornswoggles - I added the word `databases` to try to catch the SQL consideration without getting too into it

do you feel like that's enough of a distinction?

**Pull request merged**